haas/cli.py

```diff
@@ -721,15 +766,18 @@ def list_networks():
 @cmd
 def show_network(network):
     """Display information about <network>"""
-    url = object_url('network', network)
-    do_get(url)
+    q = C.project.networks_in(project)
+    sys.stdout.write("Networks allocated to {}\t:   {}\n".format(project, " ".join(q)))
 
 
 @cmd
 def show_node(node):
-    """Display information about a <node>"""
-    url = object_url('node', node)
-    do_get(url)
+    """Display information about a <node>
+    FIXME: Recursion should be implemented to the output.
```

Comment:
The output of show_node is a dictionary that can be list of list, having multiple nics and networks. 
More meta data about node could be shown via this call. So this is a note for future developers who will be making the CLI to make appropriate changes.